src/components/User/ContestsContainer.vue

```diff
@@ -22,6 +22,7 @@ defineProps<Props>()
         </router-link>
       </li>
     </ul>
+    <p v-else>実績はありません</p>
```

Comment:
ちょっと言葉が強い気がするので、↓みたいな感じで表現を緩めたいです
```suggestion
    <p v-else>まだ実績は登録されていません</p>
```